### DISCUSSION TITLE
Let’s encrypt certificate are readable only by root

### DISCUSSION THREAD

**Discussion by Geobert** [static-web-server/static-web-server#325] [Q&A] [answered]
Hi,

I’ve created a certificate and it is a directory that’s only accessible by root, but I’m using the systemd file of the repo which doesn’t run SWS as root so it can’t read certificate and private key. 
What’s the best practice here?

**Comment by joseluisq** [comment, marked as answer]
Honestly, not have a strong opinion on this. I think is up to users as anyone could handle it differently.
For example, you could symlink the certificate folder or change its permissions appropriately.
